src/Data/ProgrammesDb/Entity/RefOptions.php

```diff
@@ -23,147 +28,120 @@ class RefOptions
 
     /**
-     * @var array
+     * @var string
      *
-     * @ORM\Column(type="string", nullable=false)
+     * @ORM\Column(type="string", length=50, nullable=false)
      */
-    private $originalId;
+    private $guid;
 
     /**
-     * @var CoreEntity
+     * @var string
      *
-     * @ORM\ManyToOne(targetEntity="CoreEntity")
-     * @ORM\JoinColumn(nullable=true, onDelete="SET NULL")
+     * @ORM\Column(type="string", length=50, nullable=false)
      */
-    private $optionsForCoreEntity;
+    private $projectId;
 
     /**
-     * @var Network
+     * @var string
      *
-     * @ORM\ManyToOne(targetEntity="Network")
-     * @ORM\JoinColumn(nullable=true, onDelete="SET NULL")
+     * @ORM\Column(type="string", nullable=true)
      */
-    private $optionsForNetwork;
+    private $entity;
 
     /**
-     * @var array
+     * @var string
      *
-     * @ORM\Column(type="json_array", nullable=true)
+     * @ORM\Column(type="string", nullable=false)
      */
-    private $adminOptions;
+    private $type;
 
     /**
      * @var array
      *
      * @ORM\Column(type="json_array", nullable=true)
      */
-    private $localOptions;
-
-    /**
-     * @var string
-     *
-     * @ORM\Column(type="string", length=30, nullable=true)
-     */
-    private $projectSpace;
-
-    /**
-     * RefOptions constructor.
-     * @param string $originalId
-     * @param CoreEntity|Network $optionsFor
-     */
-    public function __construct(string $originalId, $optionsFor)
-    {
-        $this->setOriginalId($originalId);
-        $this->setOptionsFor($optionsFor);
+    private $options;
+
+    public function __construct(
+        string $entity,
+        string $guid,
+        string $projectId,
+        string $type,
+        DateTime $createdAt,
+        DateTime $updatedAt,
+        array $options = []
+    ){
+        $this->entity = $entity;
+        $this->guid = $guid;
+        $this->projectId = $projectId;
+        $this->setType($type);
+        $this->createdAt = $createdAt;
+        $this->updatedAt = $updatedAt;
+        $this->options = $options;
     }
 
-    public function getId(): ?int
+    public function getId(): int
     {
         return $this->id;
     }
 
-    public function getOriginalId(): string
+    public function getGuid(): string
     {
-        return $this->originalId;
+        return $this->guid;
     }
 
-    public function setOriginalId(string $id): void
+    public function setGuid(string $guid)
     {
-        $this->originalId = $id;
+        $this->guid = $guid;
     }
 
-    /**
-     * @return CoreEntity|Network
-     */
-    public function getOptionsFor()
+    public function getProjectId(): string
     {
-        return $this->optionsForCoreEntity ?? $this->optionsForNetwork;
+        return $this->projectId;
     }
 
-    public function getOptionsForCoreEntity(): ?CoreEntity
-    {
-        return $this->optionsForCoreEntity;
-    }
-
-    public function getOptionsForNetwork(): ?Network
+    /**
```

Comment:
This docblock doesn't convey any extra info in addition to the typehints that are already in the code so it can be removed.